### ISSUE TITLE
Add missing "clarify words"

### ISSUE THREAD

**Issue by tekul** [ThreeMinuteLearning/my3ml#4]
These weren't in the original scraped story list and are missing from the stories in the database.

**Comment by tekul** [comment]
For reference, grabbing a PHP session from the browser and then running:

``` bash
SESH=...
URL=...
END=950

for ((i=1;i<=END;i++)); do
    echo -n "$i "
    curl -b 'PHPSESSID=$SESH' $URL/story.php?p=$i | grep clarifyword | sed -E 's/\s*<input type.* readonly value="(.*)">/\1/'
done
```
does the trick. The DB commands can then be built by running

``` awk
 awk '{printf "update story set clarify_word='\''%s'\'' where id='\''%s'\'';\n", $2, $1}' 
```
on the output.

**Comment by tekul** [comment]
Fixed in 0c364793126804820
